flos/special/dneb.lua

```diff
@@ -44,4 +45,143 @@ function DNEB:neb_force(image)
    return NEB_F + PS_F - PS_F:project( P_F ) * P_F
 end
 
+
+--- Print to screen some information regarding the NEB algorithm
+function DNEB:info()
+   print("DNEB Number of Images :  " .. self.n_images)
+   print("DNEB Use Climbing After : " .. self._climbing .. " Steps")
+   local tmp = array.Array( self.n_images + 1 )
+   --tmp[1] = self:dR(0, 1):norm(0)
+   tmp[1] = self:dR(0, 1):norm(0)
+   for i = 2, self.n_images + 1 do
+      tmp[i] = tmp[i-1] + self:dR(i-1, i):norm(0)
+   end
+   print("DNEB Reaction Coordinates: ")
+   print(tostring(tmp))
+   local tmp = array.Array( self.n_images )
+   for i = 1, self.n_images do
+      tmp[i] = self.k[i]
+   end
+   print("DNEB spring constant: ")
+   print(tostring(tmp))
+end
+
+
+function DNEB:force(image, IO)
+   self:_check_image(image)
+
+   if image == 1 then
+      -- Increment step-counter
+      self.niter = self.niter + 1
+   end
+
+   local NEB_F = self:neb_force(image)
+
+   -- Things I want to output in files as control (all in 3xN format)
+   if IO then
+      local f
+
+      -- Current coordinates (ie .R)
+      f = io.open( ("DNEB.%d.R"):format(image), "a")
```

Comment:
We should discuss these. I think it is better we create a small function lua script that handles these write-outs. Makes it easier to maintain!